doc/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+.. rubric:: 1.8.1
+
+- Use :func:`np.prod` instead of deprecated :func:`np.product`.
+- Improve the exception display when accessing an unknown slot.
```

Comment:
Erm, was I meant to review that? Or was that in a separate PR?

Reply:
That was #110.